test/parallel/test-http-abort-stream-end.js

```diff
@@ -48,11 +48,13 @@ server.listen(0, () => {
       if (size > maxSize) {
         aborted = true;
```

Comment:
This can be removed, and the assertion above changed to `assert.strictEqual(req.aborted, false)`;

Reply:
Fixed